docs/src/manifolds/shapespace.md

```diff
@@ -0,0 +1,41 @@
+# Shape spaces
+
+Kendall's pre-shape and shape spaces.
+
+```@example
```

Comment:
Should we maybe add a little bit of an explanation / description and a reference to the original paper of Kendall?

Besides that I am not 100% sure whether the example would help more before or after the functions, maybe here before is fine.

Reply:
I will add some explanation and a reference.

I like adding things at the top of documentation pages because it's more visible that way. Documentation can always be found using the search function.

Reply:
Ok.